docs/en/compile.md

```diff
@@ -2,26 +2,54 @@
 
 ## Compile on Linux
 1. Download source code
-    ```
+    ```bash
     git clone git@github.com:4paradigm/OpenMLDB.git
     cd OpenMLDB
     ```
 2. Download docker image
-    ```
+    ```bash
     docker pull ghcr.io/4paradigm/hybridsql:0.4.0
     ```
 3. Start docker and map local dir in docker
-    ```
+    ```bash
     docker run -v `pwd`:/OpenMLDB -it ghcr.io/4paradigm/hybridsql:0.4.0 bash
     ```
 4. DownLoad dependencies and init env(init only once)
-    ```
+    ```bash
     cd /OpenMLDB
     bash steps/init_env.sh  
     ``` 
 5. Compile OpenMLDB
-    ```
+    ```bash
     mkdir build && cd build
     cmake ..
     make -j5 openmldb
     ```
+
```

Comment:
you probably need to add a simple section to briefly describe why we provide a customized optional Spark distribution, and what is the benefit for users. Because users has no knowledge about it. You may also refer to a blog that describes this feature if applicable.

Reply:
I will add a brief introduction about custom Spark distribution. The detailed introduction may be added in other docs so that users will learn more in other docs instead of the compile docs.

Reply:
agree